modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc
+
+[id="virt-adding-kernel-arguments-enable-IOMMU_{context}"]
+= Adding kernel arguments to enable the IOMMU driver
+
+To enable the IOMMU driver in the kernel, create the `MachineConfig` object file and add the kernel arguments.
+
+.Prerequisites
+* Administrative privilege to a working {product-title} cluster.
+* Intel or AMD CPU hardware.
+* Enable Intel Virtualization Technology for Directed I/O extensions or AMD IOMMU in the BIOS (Basic Input/Output System).
++
+[NOTE]
+====
+If you have an AMD CPU, specify the kernel argument as `amd_iommu=on`.
+====
+
+.Procedure
+. Create a `MachineConfig` object file that identifies the kernel argument. The following example shows a kernel argument for an Intel CPU.
+
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker <1>
+    name: 100-worker-iommu <2>
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+    kernelArguments:
+        - intel_iommu=on <3>
+...
+----
+<1> Applies the new kernel argument only to worker nodes.
+<2> The `name` indicates the ranking of this kernel argument (100) among the machine configs and its purpose.
+<3> Identifies the kernel argument as `intel_iommu` for an Intel CPU.
+
+. Create the new `MachineConfig` object file:
++
+[source,terminal]
+----
+$ oc create -f 100-worker-kernel-arg-iommu.yaml
```

Comment:
Consider using a generic file name here (in `<>`), because the file name doesn't have to be the one that is displayed here. This also applies to the other modules.